tests/serialise/xml-deserialise/run_siklu.py

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env python
 
-import os.path
+import os
```

Comment:
There are some code comments in these files. Do we need them?
Also, missing explanation of what this file is doing.

Reply:
actually not, i've left them only for reference...

Reply:
i'll add the description

Reply:
I think that as a rule of thumb, it is always better to remove code that is not running.